content/hi/nodes.md

```diff
@@ -0,0 +1,18 @@
+---
+title: नोड्स (Nodes) 
+status: Completed
+category: संकल्पना
+exclude_search: true
+---
+
+## यह क्या है
+
+नोड एक कंप्यूटर है जो एक सामान्य कार्य को पूरा करने के लिए अन्य कंप्यूटरों या नोड्स के साथ मिलकर काम करता है। उदाहरण के लिए, अपना लैपटॉप, मॉडेम और प्रिंटर लें। वे सभी आपके वाई-फाई (Wi-Fi) नेटवर्क से जुड़े हुए हैं जो संचार और सहयोग कर रहे हैं, प्रत्येक एक नोड का प्रतिनिधित्व करता है। क्लाउड कंप्यूटिंग में, नोड एक भौतिक कंप्यूटर, एक वर्चुअल कंप्यूटर, जिसे वर्चुअल मशीन (Virtual Machine) या एक कंटेनर भी कहा जाता है, हो सकता है।
+
+## जो समस्या यह संबोधित करता है
+
+जबकि एक आवेदन एक मशीन पर चल सकता है (और कई चलते हैं), इसमें कुछ जोखिम शामिल हैं। अर्थात् अंतर्निहित प्रणाली की विफलता आवेदन को बाधित करेगी। इसे संबोधित करने के लिए, डेवलपर्स ने डिस्ट्रीब्यूटेड एप्लिकेशन बनाना शुरू कर दिया जहां प्रत्येक प्रक्रिया अपने स्वयं के नोड पर चलती है। इस प्रकार, नोड्स एक समूह के हिस्से के रूप में ऐप्स या प्रक्रियाओं को चलाते हैं, जो एक सामान्य लक्ष्य को प्राप्त करने के लिए एक साथ काम करने वाले नोड्स के समूह, या क्लस्टर का निर्माण करते हैं।
+
+## यह कैसे मदद करता है
+
+एक नोड आपको कंप्यूट (मेमोरी, सीपीयू, नेटवर्क) की एक अलग इकाई देता है जिसे आप क्लस्टर को असाइन कर सकते हैं। क्लाउड नेटिव प्लेटफॉर्म या ऐप में एक नोड एक एकल इकाई का प्रतिनिधित्व करता है जो काम कर सकती है। आदर्श रूप से, अलग-अलग नोड्स इस मायने में अविभाज्य हैं कि किसी विशेष प्रकार का कोई एक नोड उसी प्रकार के किसी अन्य नोड से अप्रभेद्य है।
```

Comment:
अप्रभेद्य can be difficult to understand. below is a suggestion to convey the same but form sentence differently.
```suggestion
आदर्श रूप से, अलग-अलग नोड्स इस मायने में अविभाज्य हैं कि किसी विशेष प्रकार के किसी एक नोड और उसी प्रकार के किसी अन्य नोड को अलग नहीं किया जा सकता है
```